.github/workflows/github_actions.yml

```diff
@@ -249,12 +249,11 @@ jobs:
           mkdir jars
           cp ./build/libs/*.jar jars
 
-      - uses: notlmn/release-with-changelog@v2
-        name: Build_Changelog
-        with:
-          header: 'Changes:'
-          include-hash: true
-          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: mikepenz/release-changelog-builder-action@v2
+        name: Build Changelog
+        id: github_release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does this need to have access to the token?

Reply:
Oh, I guess it is probably writing to the branch?

Reply:
Yes, it is necessary according to [https://github.com/mikepenz/release-changelog-builder-action](url)